src/application/redux/actions/wallet.ts

```diff
@@ -50,14 +50,10 @@ export function setIsSpendableByMarina(accountID: AccountID, isSpendableByMarina
   };
 }
 
-export function setCustomScriptTemplate(
-  accountID: AccountID,
-  template: string,
-  changeTemplate?: string
-): AnyAction {
+export function setCustomScriptTemplate(accountID: AccountID, template: string): AnyAction {
   return {
     type: SET_CS_ACCOUNT_TEMPLATE,
-    payload: { accountID, template, changeTemplate },
+    payload: { accountID, template },
```

Comment:
why we removing changeTemplate?

Reply:
@louisinger

Reply:
`importTemplate(template: Template, changeTemplate?: Template)`

Reply:
PR https://github.com/vulpemventures/marina/pull/366 removed it

Reply:
the description of the PR is not relevant, importTemplate still have the changeTemplate param,look in master

Reply:
cc/ @louisinger

Reply:
yeah, `importTemplate` from marina provider defines a changeTemplate parameter but the the PR 366 [removed it from reducer](https://github.com/vulpemventures/marina/pull/366/files#diff-6e5548d2c1bf7cbcf1316da62161e63b60d2cde3fda5743456318ad2e2b0cf04L86) (from CustomScriptAccountData, and do not handle it anymore in CustomScriptIdentity). I've just removed it from the action payload because the reducer never use it

Reply:
ok, this seems an error then! 

if you follow the discussion, we actually agreed that is needed to have a different template for change.

If I switch to a custom account with `useAccount`, What will happen to call `getNextChangeAddress`? which kind of address will be returned and from which account?